index.js

```diff
@@ -7,12 +7,66 @@ function ask(questionText) {
   });
 }
 
+function randomInt(min, max) {
+  let range = max - min + 1;
+  return (min + Math.floor(Math.random() * range));
+}
+
+
+//Here is my sanitize function
+function saniResp(string) {
+  return string.trim().toLowerCase()
+}
+
+let lowNum = 1;
+let highNum = 100
+let randNum = randomInt(lowNum, highNum)
+
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
+  console.log("Let's play a game where you (human) make up a number between 1 and 100,\nand I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
   // Now try and complete the program.
-  process.exit();
+
+  let response = await ask("Is your number " + randNum + "? ");
+  response = saniResp(response);
+  while (response !== "yes" || response !== "y") {
+    let lastGuess = randNum
+    response = saniResp(response);
+    if (response === "yes" || response === "y") {
+      console.log("VICTORY! YOUR SOUL IS MINE!!!")
+      process.exit()
+    } else if (response === "no" || response === "n") {
+      response = await ask("Is it higher or lower? ")
```

Comment:
Here, you may want to use a hiLowResponse variable or something named differently from the response you are getting for yes or no, that way you don't get any weird behavior if a user accidentally types in 'y' or 'n' when you ask higher or lower.